File: CRUD_CSharp/Conexion.cs
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Text;

namespace CRUD_CSharp
{
    public class Conexion
    {
        public SqlConnection GetConexion()
        {
            string cadena = @"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=MyDbContext;Integrated Security=True;";
            SqlConnection con = new SqlConnection(cadena);
            con.Open();
            return con;
        }
    }
}

File: CRUD_VB/Conexion.vb
Imports System.Data.SqlClient

Public Class Conexion
    Public Function GetConexion() As SqlConnection
        Dim cadena As String = "Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=MyDbContext;Integrated Security=True;"
        Dim con As SqlConnection = New SqlConnection(cadena)
        con.Open()
        Return con
    End Function
End Class
